source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst

```diff
@@ -12,38 +12,23 @@ The installed agent runs on the endpoint you want to monitor and communicates wi
 
 #. To start the installation process, download the `HP-UX installer <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR_HPUX|/hp-ux/wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar>`_. 
 
+.. note:: You need root user privileges to run all the commands described below.
+
 #. Create the ``wazuh`` user and group.
    
    .. code-block:: console
    
        # groupadd wazuh
        # useradd -G wazuh wazuh
    
-#. Unzip the package in ``/``.
-
-   .. code-block:: console
-   
-       # tar -xvf wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar
-
-#. Edit ``/var/ossec/etc/ossec.conf`` and add the Wazuh manager IP address. The agent registers automatically with the manager through enrollment, configuration details can be found in the :doc:`Enrollment section </user-manual/reference/ossec-conf/client>`.
-
-   .. code-block:: console
-   
-      <server>
-        <address>WAZUH_MANAGER_IP</address>
-        <port>1514</port>
-        <protocol>tcp</protocol>
-      </server>
-
-
-#. To complete the installation and registration process, start the Wazuh agent service.
+#. Uncompress the package in ``/``.
```

Comment:
I think "Unzip" should be valid too

- https://www.merriam-webster.com/dictionary/uncompress

Reply:
Yes, is valid too. But uncompress is more accurate

Reply:
Ok, we will delegate it to the @wazuh/content review